src/react/components/pages/projectSettings/projectMetrics.tsx

```diff
@@ -145,12 +144,8 @@ export default class ProjectMetrics extends React.Component<IProjectMetricsProps
         const sourceAssets = await assetService.getAssets();
 
         const assetsMap = this.props.project.assets;
```

Comment:
could `project.assets` be null?

Reply:
It shouldn't be. It is always initialized to a empty object.